cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -57,9 +57,20 @@ func DefaultNameSystem() string {
 	return "public"
 }
 
+var fallbackPackages = []string{
+	"k8s.io/kubernetes/pkg/api/unversioned",
+	"k8s.io/kubernetes/pkg/apis/extensions",
+}
+
 func getInternalTypeFor(context *generator.Context, t *types.Type) (*types.Type, bool) {
 	internalPackage := filepath.Dir(t.Name.Package)
 	if !context.Universe.Package(internalPackage).Has(t.Name.Name) {
+		for _, fallbackPackage := range fallbackPackages {
```

Comment:
@lavalamp - this is also pretty ugly - do you have any idea how to make it cleaner?